src/query/src/query_engine/state.rs

```diff
@@ -84,6 +85,7 @@ impl QueryEngineState {
             analyzer.rules.insert(0, Arc::new(DistPlannerAnalyzer));
         }
         analyzer.rules.insert(0, Arc::new(TypeConversionRule));
+        analyzer.rules.insert(1, Arc::new(StringNormalizationRule));
```

Comment:
```suggestion
        analyzer.rules.insert(0, Arc::new(StringNormalizationRule));
```

They have no order dependency so just insert into the first position